DNN Platform/Library/UI/Skins/SkinController.cs

```diff
@@ -305,9 +348,12 @@ public static void SetSkin(string skinRoot, int portalId, SkinType skinType, str
                     }
 
                     break;
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(skinRoot), skinRoot, "Expected either 'Skins' or 'Containers'");

```

Comment:
I've added an exception to this method, since the method could fail silently if you made a typo. Is this OK?

Reply:
I've created a new enum called `SkinPackageType` that can be either Skin or Container.